### DISCUSSION TITLE
About keybindings, I found that `<Ctrl-Alt-p>` show a tips named "preedit".

### DISCUSSION THREAD

**Discussion by shellspicker** [zim-desktop-wiki/zim-desktop-wiki#2341] [General]
When I touch `<Ctrl-Alt-p>`, on top right corner show a tips, about preedit enabled/disabled switch, like the image shows.

![zim_preedit](https://user-images.githubusercontent.com/36255807/229319958-6b1e2224-10b1-4e5b-9859-00d51a736ae8.png)

But I didn't found that keybinding in default keybinding list, also not found that in zim help page.
So, what about that keybinds mean? It seems to no effect.

**Comment by introt** [comment]
Seems to be a toolkit (Gtk) feature related to composing special characters: https://stackoverflow.com/questions/26399614/preedit-changed-event-in-python-gtk-entry-does-not-work
